pkg/handler/module_streams_test.go

```diff
@@ -0,0 +1,161 @@
+package handler
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"io"
+	"net/http"
+	"net/http/httptest"
+	"strings"
+
+	"github.com/content-services/content-sources-backend/pkg/api"
+	"github.com/content-services/content-sources-backend/pkg/config"
+	"github.com/content-services/content-sources-backend/pkg/dao"
+	"github.com/content-services/content-sources-backend/pkg/middleware"
+	test_handler "github.com/content-services/content-sources-backend/pkg/test/handler"
+	"github.com/labstack/echo/v4"
+	echo_middleware "github.com/labstack/echo/v4/middleware"
+	"github.com/redhatinsights/platform-go-middlewares/v2/identity"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/mock"
+	"github.com/stretchr/testify/require"
+	"github.com/stretchr/testify/suite"
+)
+
+type ModuleStreamsSuite struct {
+	suite.Suite
+	echo *echo.Echo
+	dao  dao.MockDaoRegistry
+}
+
+func (suite *ModuleStreamsSuite) SetupTest() {
```

Comment:
i think you're missing the actual function to run it:


```
func TestModuleStreamsSuite(t *testing.T) {
	suite.Run(t, new(ModuleStreamsSuite))
}
```

Reply:
added!